packages/react-sdk/src/components/Whiteboard/WhiteboardHost.tsx

```diff
@@ -54,12 +54,14 @@ import { SvgCanvas } from './SvgCanvas';
 import { useWheelZoom } from './SvgCanvas/useWheelZoom';
 
 const WhiteboardHost = ({
+  contentAreaRef,
```

Comment:
This ref is passed through several components to achieve the effect to handle wheel events from the `ElementBar` additionally to `SvgCanvas`. So all "zooming" and "panning" wheel events are propagated to the same handler. From code changes it looks quite verbose currently.

From the user perspective the "panning" behavior is ok, but in the case of "zooming" it looks strange a bit. Zooming on `ElementBar` zooms the canvas. Also within current implementation this zooming behavior happens for the other toolbars: Collaboration, Tools... . 

Checking other apps it seems that a reasonable behavior could be to hide `ElementBar` when we do "zooming" and "panning" on `SvgCanvas`. And when a wheel event happens on `ElementBar` we should do prevent default to avoid browser zooming.